packages/send-type/index.js

```diff
@@ -12,7 +12,8 @@ module.exports = function (res, code=200, data='', headers={}) {
 	let type = obj[TYPE] || res.getHeader(TYPE);
 
 	if (!!data && typeof data.pipe === 'function') {
-		res.setHeader(TYPE, type || OSTREAM);
+		obj[TYPE] = type || OSTREAM;
+		res.writeHead(code, obj);
```

Comment:
Actually, scratch this – using `writeHead` works in Firefox but chokes in Chrome. Have to iterate through the headers and use `setHeader` instead.